common.blocks/i-bem-dom/i-bem-dom.js

```diff
@@ -442,7 +442,10 @@ var BemDomEntity = inherit(/** @lends BemDomEntity.prototype */{
      * @returns {Block}
      */
     findChildBlock : function(Block) {
-        // TODO: throw if Block passed as a string
+        if(typeof Block === 'string') {
+            throw new Error('Block can\'t be a String');
```

Comment:
The message which describes what type `Block` is allowed to be might be more useful in case of error. Consider "Block must be a class or an instance", as an example.